Source/WebCore/platform/network/cocoa/ResourceRequestCocoa.mm

```diff
@@ -210,7 +210,7 @@ static inline NSURLRequestCachePolicy toPlatformRequestCachePolicy(ResourceReque
 void ResourceRequest::doUpdateResourceHTTPBody()
 {
     if (NSData* bodyData = [m_nsRequest HTTPBody])
-        m_httpBody = FormData::create([bodyData bytes], [bodyData length]);
+        m_httpBody = FormData::create(std::span { static_cast<const uint8_t*>([bodyData bytes]), [bodyData length] });
```

Comment:
This is an example of where we’d use a `spanFromNSData` function.

Reply:
yes, now calling `toSpan()`